dist/stat_data.py

```diff
@@ -230,10 +230,21 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_split_leaf', 'leaf pages split during eviction'),
     CacheStat('cache_eviction_stable_state_workers', 'eviction worker thread stable number', 'no_clear'),
     CacheStat('cache_eviction_state', 'eviction state', 'no_clear,no_scale'),
+    CacheStat('cache_eviction_target_page_ge512', 'Histogram - Per tree page target greater than 512'),
+    CacheStat('cache_eviction_target_page_lt16', 'Histogram - Per tree page target 1-16'),
+    CacheStat('cache_eviction_target_page_lt32', 'Histogram - Per tree page target 16-32'),
+    CacheStat('cache_eviction_target_page_lt128', 'Histogram - Per tree page target 32-128'),
+    CacheStat('cache_eviction_target_page_lt512', 'Histogram - Per tree page target 128-512'),
     CacheStat('cache_eviction_walk', 'pages walked for eviction'),
+    CacheStat('cache_eviction_walk_from_root', 'eviction walks started from root of tree'),
+    CacheStat('cache_eviction_walk_passes', 'eviction passes of a file'),
+    CacheStat('cache_eviction_walk_saved_pos', 'eviction walks started from saved location in tree'),
     CacheStat('cache_eviction_walks_abandoned', 'eviction walks abandoned'),
     CacheStat('cache_eviction_walks_active', 'files with active eviction walks', 'no_clear,no_scale'),
+    CacheStat('cache_eviction_walks_ended', 'eviction walks reached end of tree'),
+    CacheStat('cache_eviction_walks_gave_up', 'eviction walks gave up because they saw too many pages'),
     CacheStat('cache_eviction_walks_started', 'files with new eviction walks started'),
+    CacheStat('cache_eviction_walks_stopped', 'eviction walks gave up because they walked the tree twice'),
```

Comment:
The description here isn't very accurate. No walks really walk the whole tree twice - it's an idiosyncrasy of the loop structure that the loop terminates at two restarts.